tests/components/logging/aggregate.test.js

```diff
@@ -182,6 +183,20 @@ describe('payloads', () => {
     loggingAggregate.ee.emit(LOGGING_EVENT_EMITTER_CHANNEL, [1234, Symbol('test'), {}, 'error'])
     expect(logs.pop().message).toEqual('Symbol(test)')
   })
+
+  test('initialLocation should be in pageUrl of log object attributes', async () => {
```

Comment:
This test would be more meaningful if the window.location is changed to be different from initialLocation first